src/xo-app/logs/index.js

```diff
@@ -91,41 +92,78 @@ const JobTransferredDataInfos = ({ start, end, size }) => <div>
   <span><strong>{_('jobTransferredDataSpeed')}</strong> {formatSpeed(size, end - start)}</span>
 </div>
 
-const Log = props => <ul className='list-group'>
-  {map(props.log.calls, call => {
-    const {
-      end,
-      error,
-      returnedValue,
-      start
-    } = call
+const CALL_FILTER_OPTIONS = [
+  {label: _('successfulJobCall'), value: 'success'},
+  {label: _('failedJobCall'), value: 'failed'},
+  {label: _('jobCallInProgess'), value: 'busy'},
+  {label: _('allJobCalls'), value: 'all'}
+]
 
-    let id
-    if (returnedValue != null) {
-      id = returnedValue.id
-      if (id === undefined && typeof returnedValue === 'string') {
-        id = returnedValue
-      }
-    }
+class Log extends BaseComponent {
+  state = {
+    filter: 'all'
+  }
+
+  render () {
+    const { props, state } = this
 
-    return <li key={call.callKey} className='list-group-item'>
-      <strong className='text-info'>{call.method}: </strong><JobCallStateInfos end={end} error={error} /><br />
-      {map(call.params, (value, key) => [ <JobParam id={value} paramKey={key} key={key} />, <br /> ])}
-      {end !== undefined && _.keyValue(_('jobDuration'), <FormattedDuration duration={end - start} />)}
-      {returnedValue != null && returnedValue.size !== undefined && <JobTransferredDataInfos start={start} end={end} size={returnedValue.size} />}
-      {id !== undefined && <span>{' '}<JobReturn id={id} /></span>}
-      {call.error &&
-        <span className='text-danger'>
-          <Icon icon='error' />
-          {' '}
-          {call.error.message
-            ? <strong>{call.error.message}</strong>
-            : JSON.stringify(call.error)
+    return <div>
+      <select
+        className='form-control'
+        onChange={this.linkState('filter')}
+        value={state.filter}
+      >
+        {map(CALL_FILTER_OPTIONS, ({ label, value }) => <option key={value} value={value}>{label}</option>)}
```

Comment:
You cannot use results of `_()` in a an `<option />` because `_()` returns an extra DOM element which is not supported.

Take a look: https://github.com/vatesfr/xo-web/blob/next-release/src/common/json-schema-input/enum-input.js#L49